uniffi_testing/src/lib.rs

```diff
@@ -159,7 +159,8 @@ fn get_cargo_metadata() -> Metadata {
 
 fn get_cargo_build_messages() -> Vec<Message> {
     let mut child = Command::new(env!("CARGO"))
-        .arg("build")
+        .arg("test")
+        .arg("--no-run")
```

Comment:
Could this be `build --profile=dev`?  That seems to capture the intent better to me.

Reply:
It was already `build` (without `--release`), which should've defaulted to `--profile=dev` right? `cdylibs` of [uniffi-bindgen-java](https://github.com/IronCoreLabs/uniffi-bindgen-java/blob/main/Cargo.toml#L33) weren't found on creation of the `TestHelper` as it was. `cargo build --profile=test` might work, but that feels a little less straightforward than `test --no-run` to me personally.